desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -15,11 +15,60 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+import json
+
 from django.shortcuts import render
 from django.http import HttpResponse
-from desktop.lib.django_util import login_notrequired
+from desktop.lib.django_util import login_notrequired, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+
+from django.conf import settings
+from slack_sdk import WebClient
+
+LOG = logging.getLogger(__name__)
+
+SLACK_VERIFICATION_TOKEN = getattr(settings, 'SLACK_VERIFICATION_TOKEN', None)
+SLACK_BOT_USER_TOKEN = getattr(settings, 'SLACK_BOT_USER_TOKEN', None)
 
 
+Client = WebClient(token=SLACK_BOT_USER_TOKEN)
+BOT_ID = Client.api_call('auth.test')['user_id']
+
 @login_notrequired
-def home(request):
-  return HttpResponse("Hello World")
\ No newline at end of file
+@csrf_exempt
+def slack_events(request):
```

Comment:
Time to add a views_tests.py file and add some first unit tests? e.g. for the event handling and 'hello' logic

How to mock the Client:
https://github.com/cloudera/hue/blob/master/desktop/libs/notebook/src/notebook/api_tests.py#L814
https://docs.python.org/3/library/unittest.mock.html

Reply:
Needs a first one

Reply:
Should rely start adding unit tests..

Reply:
Working on the unit tests

Reply:
Initial unit tests added